src/Tribe/Main.php

```diff
@@ -371,6 +375,9 @@ public function bind_implementations() {
 
 		// Views V2
 		tribe_register_provider( Tribe\Tickets\Events\Views\V2\Service_Provider::class );
+
+		// Promoter
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: a8bbe17a5f08984a54919e06a65c6218</sub>